backend_addon/{{ cookiecutter.__folder_name }}/.github/workflows/ci.yml

```diff
@@ -0,0 +1,128 @@
+name: {{ cookiecutter.python_package_name }} CI
+
+on:
+  push:
+
+env:
+  PYTHON_VERSION: "{{ cookiecutter.__gha_python_version }}"
+  PLONE_VERSION: "{{ cookiecutter.__gha_plone_version }}"
+
+jobs:
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+
+        - name: Checkout
+          uses: actions/checkout@{{ cookiecutter.__gha_version_checkout }}
+
+        - name: Install the latest version of uv
+          uses: astral-sh/setup-uv@{{ cookiecutter.__gha_version_setup_uv }}
+          with:
+            python-version: {{ "${{ env.PYTHON_VERSION }}" }}
+
+        - name: Check formatting
+          if: {{ "${{ success() || failure() }}" }}
+          id: ruff-format
+          run: uvx ruff@latest format --diff
+
+        - name: Check lint
+          if: {{ "${{ success() || failure() }}" }}
+          id: ruff-lint
+          run: uvx ruff@latest check --diff
+
+        - name: Check XML / ZCML
+          if: {{ "${{ success() || failure() }}" }}
+          id: zpretty
+          run: uvx zpretty@latest --check src
+
+        - name: Check Package Metadata
+          if: {{ "${{ success() || failure() }}" }}
+          id: pyroma
+          run: uvx pyroma@latest -d .
+
+        - name: Check Python Versions
+          if: {{ "${{ success() || failure() }}" }}
+          id: py-versions
+          run: uvx check-python-versions@latest .
+
```

Comment:
I don't like using `@latest` for these. That means CI can break just because a new version of the tool was released, even if there was no change in the project repository.

Reply:
My suggestion here would be to solve this in plone/meta (or some other place) and then just point the GHA files there.